docs/protocol/life_cycle_of_a_service.md

```diff
@@ -0,0 +1,41 @@
+The life cycle of a service registered in the Autonolas Protocol passes through the following states:
+
+* pre-registration,
+* active registration,
+* finished registration,
```

Comment:
Since pre-registration is written via "-", either use the same for others ("active-registration", "finished-registration"), or remove from "pre-registration" as well. I think we should follow the original FSM doc (autonolas-registries/docs/FSM.md)

Reply:
I leave @oaksprout  decide on this. I think pre- is OK, but unsure of the rest.

Reply:
fixed according to the protocol frontend. Note that Pre-Registration is capitalized in both words.